pyvista/core/dataobject.py

```diff
@@ -189,39 +189,172 @@ def copy(self, deep=True):
         Parameters
         ----------
         deep : bool, optional
-            When True makes a full copy of the object.
+            When ``True`` makes a full copy of the object.  When
+            ``False``, performs a shallow copy where the points, cell,
+            and data arrays are pointers to the original object.
 
         Returns
         -------
-        newobject : same as input
-           Deep or shallow copy of the input.
+        :class:`pyvista.DataSet`
+            Deep or shallow copy of the input.
+
+        Examples
+        --------
+        Create and make a deep copy of a PolyData object.
+
+        >>> import pyvista
+        >>> mesh_a = pyvista.Sphere()
+        >>> mesh_b = mesh_a.copy()
+        >>> mesh_a == mesh_b
+        True
 
         """
         thistype = type(self)
         newobject = thistype()
+
+        # this must be called to ensure scalars active are "active"
```

Comment:
Is this what you meant?
```suggestion
        # this must be called to ensure active scalars are "active"
```